app/client/components/form/MultipleChoice.tsx

```diff
@@ -0,0 +1,22 @@
+import Select from "react-select";
```

Comment:
Can we use ant for this instead to avoid a new package (and to begin the migration to ant)? Something like (doesn't work u get the point):
```
import React from "react";
import { Select } from "antd";
const { Option } = Select;

export const MultipleChoice = (props: any) => {
  const { name, input, options } = props;


  const parsedOptions = options.map((option: string) => (
    <Option key={option} value={option}>
      {option}
    </Option>
  ));

  return (
    <Select
      {...input}
      defaultValue={options[0]}
      onChange={value => input.onChange(value)}
      onBlur={() => input.onBlur(input.value)}
    >
      { parsedOptions }
    </Select>
  );
};
```